posthog/api/test/__snapshots__/test_query.ambr

```diff
@@ -237,9 +237,38 @@
   /* user_id:0 request:_snapshot_ */
   SELECT event,
          events.distinct_id,
-         replaceRegexpAll(JSONExtractRaw(events.properties, 'key'), '^"|"$', ''),
+         replaceRegexpAll(JSONExtractRaw(properties, 'key'), '^"|"$', ''),
```

Comment:
Previously `events.` was added before `properties` to disambiguate it from the `events__pdi__person.properties` column that we selected in the join. That's no longer the case and thus it seems like it was automatically removed.